### ISSUE TITLE
Feature request: support new ARIA Annotations from ARIA 1.3

### ISSUE THREAD

**Issue by aleventhal** [dequelabs/axe-core#2078]
ARIA Annotations includes the following new markup:
Roles: insertion, deletion, mark, suggestion comment
Properties: aria-description
In addition, aria-details now supports multiple ids
Here is an explainer: https://github.com/aleventhal/aria-annotations
All of the new markup has landed in the ARIA draft master.

Note that there are a number of places in axe-core that look at aria-describedby, that should also look at aria-description.

Motivation: Chrome uses axe-core for some internal tests, and the lack of support for aria-description is preventing a simplification change in devtools.

**Comment by straker** [comment]
Thanks for the issue. Do you know if any browser/screen reader combination supports any of these features? I tested `aria-description` but it didn't seem VO/Mac, JAWS/IE, JAWS/Chrome, nor NVDA/Firefox  recognize it.

**Comment by aleventhal** [comment]
Thanks for replying @straker .
It's in Chrome 82, meaning that you have to use Canary builds to see it for now. Chrome 82 will become beta in March 26 and stable March 28.

In the meantime if you use Chrome 82, it will work with any screen reader, because it uses the same API methods to expose the description to screen readers as aria-describedby.

**Comment by aleventhal** [comment]
More info:
- It's not supported in Firefox yet (https://bugzilla.mozilla.org/show_bug.cgi?id=1608975), but it is marked as "important" in their public standards positions (https://mozilla.github.io/standards-positions/#aria-annotations).
- Because it's supported in Chrome 82, it will automatically be supported in the new Edge.
- No public signals from Apple yet, but there is an issue filed here: https://bugs.webkit.org/show_bug.cgi?id=206253

**Comment by straker** [comment]
Thanks for all those links. So I believe that since it's not supported in anything but Chrome canary this wouldn't fall under [accessibility support](https://github.com/dequelabs/axe-core/blob/develop/doc/accessibility-supported.md#accessibility-supported) so I'm unsure if we could support it at this time. 

However @WilcoFiers would be able to clarify how to move forward with this.

**Comment by aleventhal** [comment]
Thanks for linking to accessibility supported. It makes sense as a general principle, but I wonder if the list should be modernized. For example, why have IE in there at this point, and yet the only Chrome combo is with Talkback?

Chrome supports NVDA, JAWS, ChromeVox, Orca and VoiceOver. Like the new Edge, it will soon support Narrator.

**Comment by WilcoFiers** [comment]
There are a couple of topics to address here, so I'll just take 'm one at a time.

- New roles from ARIA 1.2; I think we can add those once ARIA 1.2 makes it to candidate recommendation. Those new roles won't get in the way of users if AT don't support them, so I see no reason to hold on that.

- Adding ARIA 1.3 features; this seems premature. The first public working draft for ARIA 1.3 has not yet been published. I don't think we should add them in at this point, but I totally agree with your use case here. Things in Chrome only have to work for AT that support Chrome, so that's a subset of axe-core's baseline. The aria-allowed-attr check can be configured to allow unknown attributes for specific roles. There is no option for global attributes in there, but that's pretty easy to add.

- That acc support list needs to be updated for the next major release of axe-core (planned for some time later this year). It's definitely a little dated.

**Comment by aleventhal** [comment]
Thanks, this is all helpful. Should we open a configuration feature request to allow specific global attributes?

**Comment by WilcoFiers** [comment]
Done, closing this ticket as I think everything that needs to be discussed here has been.
